lib/util/util.js

```diff
@@ -50,6 +50,31 @@ const Util = {
     standardizedSplitChar: '/',
     skipInteraction: false,
 
+    /**
+     * helper to quickly ensure we are working with a flat Array of items
+     * @param {MetadataTypeMap|MetadataTypeItem[]} metadata map or array of metadata
+     * @returns {MetadataTypeItem[]} Array of metadata items
+     */
+    getMetadataMapToList(metadata) {
+        if (metadata && Array.isArray(metadata)) {
+            return metadata;
+        } else {
+            return Object.keys(metadata).map((item) => metadata[item]);
```

Comment:
I think you mean to use: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/values
Also, I would wonder if this is really the right approach? We should always know what we are returning, and if this is needed it shows there is bugs which need fixing